mama/c_cpp/src/c/SConscript

```diff
@@ -183,18 +183,26 @@ else:
 
 fp.close()
 
-objects = []
+static_objects = []
+shared_objects = []
 StaticLibs['libmama'] = []
 
 
 for s in libmama_sources:
-    obj = env.Object(s)
-    StaticLibs['libmama'].append(obj)
-    objects.append(obj)
+    if s == "msg.c":
+        static_obj = env.Object(s, CFLAGS="-Wno-deprecated-declarations")
```

Comment:
can we stick a comment in here explaining why we can't use our "ignore deprecated" macros?